src/Orchard.Web/Modules/Orchard.Autoroute/ImportExport/HomeAliasImportHandler.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Linq;
+using System.Web.Routing;
+using Orchard.Autoroute.Services;
+using Orchard.ContentManagement;
+using Orchard.Recipes.Models;
+using Orchard.Recipes.Services;
+
+namespace Orchard.Autoroute.ImportExport {
+    public class HomeAliasImportHandler : Component, IRecipeHandler {
+        private readonly IHomeAliasService _homeAliasService;
+        private readonly IContentManager _contentManager;
+
+        public HomeAliasImportHandler(IHomeAliasService homeAliasService, IContentManager contentManager) {
+            _homeAliasService = homeAliasService;
+            _contentManager = contentManager;
+        }
+
+        public void ExecuteRecipeStep(RecipeContext recipeContext) {
+            if (!String.Equals(recipeContext.RecipeStep.Name, "HomeAlias", StringComparison.OrdinalIgnoreCase)) {
```

Comment:
I think it's pretty clear that opinions on this matter differ. What we need to do is make a decision and stick with it, Personally I am in favor of using the .NET type when invoking static members, but I am even more for sticking with coding guidelines established by the ASP.NET team, as it is likely that others will follow that as well, resulting in a more global standard, which is a good thing. Let's discuss this on tomorrow's call and create a poll or something, then make a final decision.